### PR TITLE
Use relative path for docroot on mkdocs

### LINKED ISSUE #1917: Flyout links incorrect on Mkdocs projects
GitHub links in the sidebar footer contain an extraneous path. For example, on http://tracks.readthedocs.org/en/latest/upgrading/ the "View" link points to https://github.com/TracksApp/tracks/blob/master/home/docs/checkouts/readthedocs.org/user_builds/tracks/checkouts/latest/doc/Upgrading.md.

### PR DESCRIPTION
**Pull request by stsewd**
While replicating #3515, I noticed that the other "edit" link (at the float version menu) was wrong https://github.com/rtfd/readthedocs.org/issues/3515#issuecomment-358110500

Fixes #1917

### CODE REVIEW

**Comment by stsewd** [comment]
Due to #3497, with this change the first build fails, probably we want to solve that issue before merging.

**Comment by marcelstoer** [comment]
@RichardLitt maybe add the Mkdocs label as well?

**Comment by stsewd** [comment]
The origaal blocking issue https://github.com/rtfd/readthedocs.org/pull/3525#issuecomment-358542018 can't be replicated, so it's safe to continue, I only need to add a test, but I started some tests for mkdocs in https://github.com/rtfd/readthedocs.org/pull/4184/, so I would like to merge that PR first.

File: readthedocs/doc_builder/backends/mkdocs.py [hunk 1]
```diff
@@ -119,11 +119,16 @@ def append_conf(self, **__):
             '%scss/readthedocs-doc-embed.css' % media_url,
         ])
 
-        docs_path = os.path.join(self.root_path, docs_dir)
+        # The docs path is relative to the location
+        # of the mkdocs configuration file.
+        docs_path = os.path.join(
```
Comment (stsewd) [inline readthedocs/doc_builder/backends/mkdocs.py]: https://www.mkdocs.org/user-guide/configuration/#docs_dir

**Comment by humitos** [comment]
@stsewd Is it needed something else here to merge this PR? If no, feel free to merge.

**Comment by stsewd** [comment]
We are all green again

**Pull request merged**